src/transport/TransportFactory.cpp

```diff
@@ -46,6 +46,7 @@ TransportFactory::TransportFactory()
     addDeprecatedAlias("water", "Water");
     reg("high-pressure", []() { return new HighPressureGasTransport(); });
     addDeprecatedAlias("high-pressure", "HighP");
+    reg("high-pressure-chung", []() { return new ChungHighPressureGasTransport(); });
```

Comment:
Should we consider giving the `high-pressure` model a more specific name (and if so, what would it be)? From my reading of the Poling book, it's not clear to me what set of models you'd want as a "default".

Reply:
This is definitely open to suggestions. The high-pressure model is a selection of 3 different models lumped together. Lucas for the viscosity, Takahashi for the diffusion coefficient, and that Ely and Hanley method for the thermal conductivity. It's hard to slap a name on that.  With the Chung class, at least it's 2/3rd of Chung's work, so the name makes a little more sense, but even that might be unclear because it's using Takahashi's method for diffusion coefficinets (there is no Chung method in the Poling book for diffusion coefficients sadly).

Reply:
Maybe high-pressure-LET and high-pressure-CCT, or high-pressure-LTE and high-pressure-CT  Where we just use the first letter of the model name for the viscosity, thermal conductivity, and diffusion coefficient.

The transport class is essentially 3 smaller transport classes in a trench coat.

Reply:
"3 smaller transport classes in a trench coat" is a very apt description. In the long run (but definitely not right now), it might be worth thinking of a different code structure that was more amenable to mixing and matching the individual models. I'm OK with just using `high-pressure` and `high-pressure-Chung` (note the capitalization) as the names for now.